### PR TITLE
Add a chapter for file and directory sharing

### LINKED ISSUE #26: Add a chapter for file and directory sharing between container and host
While directory binding is briefly mentioned while running interactive shells, it is useful and relevant to expand the discussion on how to share files and directories with the container in a dedicated chapter. 

Some information may be taken from https://apptainer.org/docs/user/main/bind_paths_and_mounts.html

### PR DESCRIPTION
**Pull request by michmx**
Explaining directory binds between the host and the container in a new chapter.

### CODE REVIEW

File: _episodes/07-file-sharing.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Remember that in Singularity, your user outside is the same inside the container. And the same happens with permissions and
+ownership for files in bind directories.
+
+## Bind paths included by default
+
+For each container executed, Singularity binds automatically some directories by default, and other defined the system
+admin in the Singularity configuration. By default, Singularity binds:
+* The user's home directory ($HOME)
+* The current directory when the container is executed ($PWD)
+* System-defined paths: ``/tmp``, ``/proc``, ``/dev``, etc.
```
Comment (klieret) [inline _episodes/07-file-sharing.md]: (Someone's been using too much sphinx ;)   It'll probably work, but standard markdown has single ticks for code.)

File: _episodes/07-file-sharing.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+The command-line option ``--bind (-B)`` will specify the directories that must be linked between the
+host and the container. It is available for ``run``, ``exec`` and ``shell`` (as well for ``instance`` that is
+not covered yet).
+
+The syntax for using the bind option is ``"source:destination"``, and the paths must be absolute (relative
+paths will be rejected). For example
+~~~bash
+singularity shell --bind /home/user/mydata:/data my_container.sif
+~~~
+{: .source}
```
Comment (klieret) [inline _episodes/07-file-sharing.md]: this mixture of markup seems strange to me, shouldn't it be

```
~~~
...
~~~
{: .language-bash}
```

or simply 

    ```bash

File: _episodes/07-file-sharing.md (hunk 1 above)
Comment (michmx) [reply _episodes/07-file-sharing.md]: Haha, oh yes! Documentation, documentation everywhere....

**Review by klieret — APPROVED** [review]
I'm wondering if there's a way to add exercises to this or to tie it in with the rest of the lesson (I have no idea, but probably we're building up a bigger example, right)?

File: _episodes/07-file-sharing.md (hunk 2 above)
Comment (michmx) [reply _episodes/07-file-sharing.md]: You are right, and looks cleaner. Let me replace all, ensure that I didn't break anything and push.

**Pull request merged**